src/lib.rs

```diff
@@ -127,8 +127,8 @@ pub fn children_with_depth(i: u64, depth: u64) -> Option<(u64, u64)> {
   } else if depth == 0 {
     Some((i, i))
   } else {
-    let off = offset_with_depth(i, depth) >> 1;
-    Some((index(depth - 1, off), index(depth - 1, off + 1)))
+    let offset = offset_with_depth(i, depth) * 2;
```

Comment:
`<< 2` would also do the job :)

Reply:
Oh sick! Yeah, I gotta brush up on my binary operators lolol.

Reply:
Oh no, that outputs `(16, 18)` instead of `(8, 10)`. Not sure it's quite the same then :/